### ISSUE TITLE
`integration` example fails to run with a validation error

### ISSUE THREAD

**Issue by tasgon** [hecrj/iced#775]
After running `cargo run --package integration`, I get this:

```
wgpu error: Validation Error

Caused by:
    In a RenderPass
      note: encoder = `<CommandBuffer-(0, 1, Vulkan)>`
    In a set_scissor_rect command
    Invalid ScissorRect parameters


thread 'main' panicked at 'Handling wgpu errors as fatal by default', /home/<user>/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.0/src/backend/direct.rs:1896:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by thenlevy** [comment]
I managed to reproduce this. I think the problem is the same as in #816, because I needed to adjust my graphical setting so that the value of `scale factor` was 1.75

Also I got a similar backtrace as in #816, the validation error occurs in `iced_wgpu::Backend::flush` when a call to `RenderPass::set_scissor_rect` is made

**Comment by kaimast** [comment]
Can reproduce this too.

**Comment by hecrj** [comment]
Fixed by #818.
